ghost/i18n/locales/sl/newsletter.json

```diff
@@ -1,24 +1,24 @@
 {
-    "By {authors}": "",
-    "Comment": "",
-    "complimentary": "",
-    "Email": "",
-    "free": "",
-    "Keep reading": "",
-    "Less like this": "",
-    "Manage subscription": "",
-    "Member since": "",
-    "More like this": "",
-    "Name": "",
-    "paid": "",
-    "Subscription details": "",
-    "trialing": "",
-    "Unsubscribe": "",
-    "View in browser": "",
-    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "",
-    "Your free trial ends on {date}, at which time you will be charged the regular price. You can always cancel before then.": "",
-    "Your subscription has been canceled and will expire on {date}. You can resume your subscription via your account settings.": "",
-    "Your subscription has expired.": "",
-    "Your subscription will expire on {date}.": "",
-    "Your subscription will renew on {date}.": ""
+    "By {authors}": "Avtorji: {authors}",
+    "Comment": "Komentar",
+    "complimentary": "promocijsko",
+    "Email": "E-pošta",
+    "free": "brezplačno",
+    "Keep reading": "Berite naprej",
+    "Less like this": "Manj takšnih vsebin",
+    "Manage subscription": "Upravljanje naročnine",
+    "Member since": "Član od",
+    "More like this": "Več takšnih vsebin",
+    "Name": "Ime",
+    "paid": "plačano",
+    "Subscription details": "Podrobnosti naročnine",
+    "trialing": "poskusna uporaba",
+    "Unsubscribe": "Odjavite se",
+    "View in browser": "Oglejte si v brskalniku",
+    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "To prejemate, ker ste <strong>%%{status}%% naročnik</strong> na {site}.",
```

Comment:
Status will be substituted into this.  Possible status are your translations of free/complimentary/paid/trialing.  Could you please check that grammatically correct sentences result when this substitution occurs?  
@coderabbitai - this would be a good thing to check for on future translations of newsletter.json, as it's been a common problem.